WalletWasabi.Gui/Controls/WalletExplorer/CoinListViewModel.cs

```diff
@@ -138,47 +138,27 @@ private void RefreshOrdering()
 		{
 			if (AmountSortDirection != SortOrder.None)
 			{
-				if (AmountSortDirection == SortOrder.Increasing)
-				{
-					MyComparer = SortExpressionComparer<CoinViewModel>.Ascending(cvm => cvm.Amount);
-				}
-				else
-				{
-					MyComparer = SortExpressionComparer<CoinViewModel>.Descending(cvm => cvm.Amount);
-				}
+				MyComparer = AmountSortDirection == SortOrder.Increasing
+					? SortExpressionComparer<CoinViewModel>.Ascending(cvm => cvm.Amount) 
```

Comment:
You could elevate `SortExpressionComparer<CoinViewModel>` out to a variable.

Reply:
Done.